src/sql/workbench/contrib/assessment/browser/media/asmt.css

```diff
@@ -0,0 +1,248 @@
+.asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row .slick-cell>.carbon-taskbar.error-val>.codicon,
+.asmtview-grid .detail-container .error-val {
+	width: 100%;
+	opacity: 1;
+	font-weight: 700;
+	text-overflow: ellipsis;
+	color: orangered;
+}
+
+.asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row .slick-cell>.warning-val,
+.asmtview-grid .detail-container .warning-val {
+	width: 100%;
+	opacity: 1;
+	font-weight: 700;
+	text-overflow: ellipsis;
+	color: darkgoldenrod;
+}
+
+
+.asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row .slick-cell._detail_selector.error-row {
+	opacity: 1;
+}
+
+#asmtDiv .detail-container {
+	max-height: 100px !important;
+	line-height: 20px;
+}
+
+#asmtDiv .detail {
+	padding: 5px;
+}
+
+#asmtDiv .preload {
+	font-size: 13px;
+}
+
+#asmtDiv .dynamic-cell-detail> :first-child {
+	vertical-align: middle;
+	line-height: 13px;
+	padding: 10px;
+	margin-left: 20px;
+}
+
+
+asmtview-component .asmtview-grid .grid-canvas>.ui-widget-content.slick-row>.slick-cell {
+	cursor: pointer;
+}
+
+.vs-dark .asmtview-grid>.monaco-table .slick-header-columns .slick-resizable-handle {
+	border-left: 1px dotted white;
+}
+
+.hc-black .asmtview-grid>.monaco-table .slick-header-columns .slick-resizable-handle {
+	border-left: 1px dotted #2b56f2;
+}
+
+.asmtview-grid>.monaco-table .slick-header-columns .slick-resizable-handle {
+	border-left: 1px dotted #444444;
+}
+
+#asmtDiv .codicon.in-progress {
+	padding-left: 0px;
+}
+
+#asmtDiv .carbon-taskbar .action-item {
+	margin-left: 0px;
+}
+
+#asmtDiv .asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row:hover>.slick-cell,
+#asmtDiv .asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row.hovered>.slick-cell,
+#asmtDiv .asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row>.slick-cell.hovered {
+	background: #dcdcdc !important;
+}
+
+.vs-dark #asmtDiv .asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row:hover>.slick-cell,
+.vs-dark #asmtDiv .asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row>.slick-cell.hovered,
+.vs-dark #asmtDiv .asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row.hovered>.slick-cell {
+	background: #444444 !important;
+}
+
+.hc-black #asmtDiv .asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row:hover>.slick-cell,
+.hc-black #asmtDiv .asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row>.slick-cell.hovered,
+.hc-black #asmtDiv .asmtview-grid>.monaco-table .slick-viewport>.grid-canvas>.ui-widget-content.slick-row.hovered>.slick-cell {
+	background: none !important;
+}
+
+.vs .action-label.codicon.refreshIcon {
```

Comment:
We already have a refresh icon defined here which you should try to use instead https://github.com/microsoft/azuredatastudio/blob/master/src/sql/media/icons/common-icons.css#L176

Reply:
The same goes for the other icons - use the common ones when you can and if you have new ones and think they're generically useful enough you can add them there

Reply:
I have replaced my icons for server and database with ones from the list you've provided. There are still some icons that I can't replace with the common ones. This is because they are in blue color, while I need them in grayscale (since all icons on the assessment tab are in grayscale).
I'm not sure if I need to upload a monochrome version of these two icons (configure-dashboard and new-query) in the common-icons set. Please advise.

Reply:
Where are you getting your icons from? In general we should try to align icon styles across the product and we've been migrating to using the blue icons ourselves so it'd be ideal if you were able to get similar ones made for your stuff.

Reply:
So, we have an action bar with icons. There could be Server or Database icons, "Export as Script" and "Learn more at Github". The first two are monochrome and are taken from common-icons set, but the other two I had to convert to monochrome (just changed color in svg file) and put in \assessment\browser\media folder.  So, we either need server and database icons in the common-set to be blue or have "configure-dashboard" and "new-query" to be grayscale.

Reply:
@Charles-Gagnon we understand that the icons should be revised. Is this a blocker at the current stage? I mean can we work out new icons in the next version?